src/__Tests__/calculate.test.js

```diff
@@ -0,0 +1,211 @@
+import calculate from '../logic/calculate';
+
```

Comment:
- [ ] One test suite is failing. Kindly ensure to add some test cases in the `App.test.js` file, or remove where appropriate.

**Failing test suite:**
![image](https://user-images.githubusercontent.com/12745474/141353996-8e279991-8463-4cb4-a493-035f20b33908.png)